### PR TITLE
Calculator setup

### PR DESCRIPTION
**Pull request by melaniesigrid**
Thank you for reviewing my project! On this PR...

- Removed and adapted boilerplate code
- Created calculator class
- Styled calculator according to medium-fidelity wireframe

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,9 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```
Comment (rustam-tolipov) [inline src/App.js]: - [x] Good job so far 👏👍. Please kindly make the `App.js` class-based like you used in `Calculator.js` component. Good luck 🍀

File: src/components/calculator.css [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.calculator-container {
+  background-color: #1a1a1a;
+  color: white;
+  width: 30rem;
+  height: 30rem;
+  display: grid;
+  grid-template-rows: 1fr 4fr 1fr;
+}
```
Comment (rustam-tolipov) [inline src/components/calculator.css:8]: - [x] [OPTIONAL] Your calculator looks nice as in the given design 👍. I'd suggest you put it to the center of the page to make it better 💡

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @melaniesigrid , 

Good job so far!

### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯


There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (melaniesigrid) [reply src/App.js]: Thank you for your review! I have made App class based.

File: src/components/calculator.css (hunk 2 above)
Comment (melaniesigrid) [reply src/components/calculator.css:8]: Thank you for your review! I have centered the calculator using flexbox.

**Review by bimbolabuari — APPROVED** [review]
# Project Approved 🚀

Hi @melaniesigrid🥇,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**